PanoptesNetClient/PanoptesNetClient/ApiClient.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace PanoptesNetClient
+{
+    public class ApiClient
+    {
+        static HttpClient client = new HttpClient();
+        private static ApiClient instance;
+
+        private ApiClient()
+        {
+            RunAsync().GetAwaiter().GetResult();
+        }
+
+        public static ApiClient Instance
+        {
+            get
+            {
+                if (instance == null)
+                {
+                    instance = new ApiClient();
+                }
+                return instance;
+            }
+        }
+
+        static async Task RunAsync()
+        {
+            client.BaseAddress = new Uri(Config.Host);
+            client.DefaultRequestHeaders.Accept.Clear();
+            client.DefaultRequestHeaders.Accept.Add(
+                new MediaTypeWithQualityHeaderValue("application/json"));
+            client.DefaultRequestHeaders.Add("Accept", "application/vnd.api+json; version=1");
+            try
+            {
+                await GetProjectAsync();
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e.Message);
+            }
+        }
+
+        static async Task<JObject> GetProjectAsync()
```

Comment:
I don't think this should be `static`

Reply:
I don't know if this is a C# naming thing, but I think it's a little redundant to add Async to the name when `async` is in the function definition.

Reply:
Nah, it's not a convention to have it in the name (`GetProjectAsync`). However, the `async` does need to be in the definition.

Reply:
Actually, turns out I did name this based on advice from this article https://docs.microsoft.com/en-us/aspnet/web-api/overview/advanced/calling-a-web-api-from-a-net-client#sending-a-post-request-to-create-a-resource